app-crypt/sbctl/files/sbctl-0.14-installkernel-bashism.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/contrib/kernel-install/91-sbctl.install b/contrib/kernel-install/91-sbctl.install
```

Comment:
Thanks! Can you add upstream/downstream references according to https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#how?

Reply:
I hope this is the expected form for the patch